flask_apispec/extension.py

```diff
@@ -34,9 +33,13 @@ def get_pet(pet_id):
 
     :param Flask app: App associated with API documentation
     :param APISpec spec: apispec specification associated with API documentation
+    :param enable_auth_key: Should we include a authentication token field in swagger ui
+    :param auth_key_name: The name of the auth token to use, defaults to 'auth_token'
```

Comment:
Wouldn't it be better to combine those two parameters in a single `auth_key` with a default to `None`,  and the auth key name otherwise, as to not increase the surface of the public API  too much? Not sure. It could still be used as `{% if auth_key %}` etc.